### ISSUE TITLE
feat: add home button to mobile

### ISSUE THREAD

**Issue by matthewkurtti** [ourjapanlife/findadoc-web#1501]
# Problem
Currently, there is a home button in the desktop navbar to navigate back to the main page, but this is missing in the mobile hamburger menu

# Success

Add this to the mobile hamburger menu

### Requirements

- [x] user can navigate back to home page (without manually typing in url)
- [x] home button is in the mobile hamburger menu
- [x] #1515

# Implementation notes

**Comment by Tch4lla** [comment]
I can take this issue. Is there a specific design for the home button?

**Comment by Tch4lla** [comment]
Currently the navbar covers the search stuff on the left. I will handle the optional task in a different issue, after the nav and search bar are both visible simultaneously
